modules/Bio/EnsEMBL/Variation/Pipeline/ProteinFunction/RunSift.pm

```diff
@@ -136,6 +137,18 @@ sub run {
         $alignment_ok = 1;
       }
       else {
+        # If we just did not find enough hits, that's ok, just skip this
+        my $error_file = $fasta_file . ".query.globalX.error";
+        if (-s $error_file) {
+          open my $error_fh, "<", $error_file or die $!;
+          my $error_msg = <$error_fh>;
+          close $error_fh;
+          chomp $error_msg;
+          if ($error_msg =~ /Not enough sequences \(only \d\) found by the PSI-BLAST search!|PSI-BLAST found no hits/) {
+            return;
```

Comment:
We still want to get this information , not enough sequence , somewhere. Can that be written as a warning message into the log table? Let's have someone else's opinion too.